rotors_gazebo_plugins/include/rotors_gazebo_plugins/gazebo_bag_plugin.h

```diff
@@ -32,38 +32,48 @@
 #include <geometry_msgs/PoseStamped.h>
 #include <geometry_msgs/TwistStamped.h>
 #include <geometry_msgs/WrenchStamped.h>
-#include <mav_msgs/CommandAttitudeThrust.h>
-#include <mav_msgs/CommandMotorSpeed.h>
-#include <mav_msgs/CommandRateThrust.h>
-#include <mav_msgs/CommandTrajectoryPositionYaw.h>
+#include <mav_msgs/Actuators.h>
+#include <mav_msgs/AttitudeThrust.h>
+#include <mav_msgs/default_topics.h>
+#include <mav_msgs/RateThrust.h>
 #include <ros/ros.h>
 #include <rosbag/bag.h>
 #include <sensor_msgs/Imu.h>
 #include <std_msgs/Float32.h>
+#include <trajectory_msgs/MultiDOFJointTrajectoryPoint.h>
 
 #include "rotors_gazebo_plugins/common.h"
 
 
 namespace gazebo {
-// Default values
+// Default values, as many of these from mav_msgs as possible.
 static const std::string kDefaultNamespace = "";
 
 static const std::string kDefaultGroundTruthPosePubTopic = "ground_truth/pose";
 static const std::string kDefaultGroundTruthTwistPubTopic = "ground_truth/twist";
-static const std::string kDefaultImuPubTopic = "imu";
-static const std::string kDefaultImuSubTopic = "imu";
-static const std::string kDefaultControlAttitudeThrustPubTopic = "command/attitude";
-static const std::string kDefaultControlAttitudeThrustSubTopic = "command/attitude";
-static const std::string kDefaultControlMotorSpeedPubTopic = "command/motor_speed";
-static const std::string kDefaultControlMotorSpeedSubTopic = "command/motor_speed";
-static const std::string kDefaultControlRateThrustPubTopic = "command/rate";
-static const std::string kDefaultControlRateThrustSubTopic = "command/rate";
-static const std::string kDefaultMotorPubTopic = "motors";
+static const std::string kDefaultImuPubTopic = mav_msgs::default_topics::IMU; // "imu"
```

Comment:
let's decide where we put this stuff ... these are also defined in rotors_gazebo_plugins/common.h I thought. I'd go for using rotors_gazebo_plugins/common.h, or drop even that and use mav_msgs/default_topics.h. less maintenance.